exemption_descriptions.yml

```diff
@@ -1,5 +1,10 @@
 ---
-description:
-    'disclosure-transparency-rules-chapter-5-applies' : "From {exempt_from} the company is exempt from keeping a PSC register because it’s a DTR issuer and the shares are admitted to trading on a regulated market"
+exemption_type:
+    'disclosure-transparency-rules-chapter-five-applies' : "The company has been or is exempt from keeping a PSC register, because it is a DTR issuer and the shares are admitted to trading on a regulated market."
```

Comment:
I fully agree with the above but in this particular case the api released was using the new enumeration values, so any coding made with the old values would not have worked however the data set in question was introduced to the SBEE act, any there was no data at the time. Unfortunately the values were not merged down in to master.

We are conscious of the versioning implications and we will be tying enumeration versions to our api versions when we release our versioning scheme; until then we cannot make any breaking changes.